packages/core/src/components/Dropdown/Dropdown.example.html

```diff
@@ -1,4 +1,6 @@
-<label class="ds-c-label ds-u-margin-top--0" for="options">Dropdown label</label>
+<label class="ds-c-label ds-u-margin-top--0" for="options"
+  >Dropdown label</label
```

Comment:
This is another weird thing that prettier does... I've never seen this formatting before

Reply:
I think it might be fixed with some new lines though. If that works I'll commit it